Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python \
+      python-pip \
+      python-dev \
+      python-setuptools \
+      ca-certificates \
+      gcc \
+      libpq-dev \
+      curl \
+      gunicorn \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# kelseyhightower confd arm64 builds are broken: https://github.com/kelseyhightower/confd/issues/850
+ARG CONFD_VERSION=0.19.1
+RUN CONFD_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) \
+    && curl -SL "https://github.com/abtreece/confd/releases/download/v${CONFD_VERSION}/confd-v${CONFD_VERSION}-linux-${CONFD_ARCH}.tar.gz" | tar -xz -C /usr/local/bin/ \
+    && /usr/local/bin/confd -version
+
+RUN groupadd --system --gid 510 osqa && useradd --system --gid osqa --uid 510 osqa
```

Comment:
Does a specific UID/GID matter once we're in a container?

Reply:
It is still recommended not to run apps as root within a root container. 

The UID / GID will be for compatibility later with existing OSQA user uploads when I enable a docker volume mount to store the uploads outside the container.

Reply:
I wasn't suggesting using root inside the container, just letting useradd pick a random ID but sharing with the host is indeed a good reason to fix it.